mlflow/utils/server_cli_utils.py

```diff
@@ -0,0 +1,58 @@
+"""
+Utilities for MLflow cli server config validation and resolving.
+
+NOTE: these functions are intended to be used as utilities for the cli click-based interface.
```

Comment:
@dbczumar refactored the module name and added some warnings in here about not using this outside of cli interfaces.